docs/docsite/_ext/ansible_redirects.py

```diff
@@ -0,0 +1,27 @@
+"""Sphinx extension for generating redirects in Ansible package docs."""
+
+import os
+from pathlib import Path
+
+import yaml
+from sphinx.application import Sphinx
+
+DOCSITE_ROOT_DIR = Path(__file__).parents[1].resolve()
+ANSIBLE_REDIRECTS_CONFIG_PATH = DOCSITE_ROOT_DIR / "ansible_redirects.yaml"
```

Comment:
With more extensions coming in, would it make sense to create a subdir for configs?

Reply:
I think anything we can do to logically organize things is good. Do you mean an subdir for extension config only or more in a general sense?